docs/src/integrations/observability/blackfire.md

```diff
@@ -31,75 +31,9 @@ Go to your Dashboard and create a new environment [under the Environments tab](h
 
-You will need to store the server credentials for further configuration. You can find them any time under the "Settings" tab of your environment in Blackfire.
-
-![Blackfire credentials](/images/integrations/blackfire/blackfire-credentials.png "0.4")
-
-### 2. Enable the Blackfire extension
-
-Configure the extension in your `.platform.app.yaml` as follows:
-
-```yaml
-runtime:
-    extensions:
-        - blackfire
-```
-
-Push the changes to your Platform environment to enable Blackfire as follows:
-
-```bash
-git add .platform.app.yaml
-git commit -m "Enable Blackfire."
-git push
-```
-
-### 3. Configure your server credentials
-
-Blackfire enables to have a fine grained configuration of server credentials across branches and environments on Platform.sh.
-
-#### Configuring global server credentials
-
-Configuring server credentials on your master branch will enable you to make sure you can profile any other branch:
-
-```bash
-platform variable:create -e master --json=false --visible-build=false --sensitive=false --level=project env:BLACKFIRE_SERVER_ID --value <insert your Server ID>
-platform variable:create -e master --json=false --visible-build=false --sensitive=false --level=project env:BLACKFIRE_SERVER_TOKEN --value <insert your Server Token>
-```
-
-#### Configuring server credentials per branch
-
-A recommendation is to have a [Blackfire environment](https://blackfire.io/docs/reference-guide/environments#documentation) for production, another one for staging, and another one for development/integration. That can be mapped in Platform.sh to one Blackfire environment for the production branch, one for the staging branch, and one for all feature branches.
-
-
-```bash
-platform variable:create -e=<insert your branch name> --json=false --visible-build=false --sensitive=false --level=environment env:BLACKFIRE_SERVER_ID --value <insert your Server ID>
-platform variable:create -e=<insert your branch name> --json=false --visible-build=false --sensitive=false --level=environment env:BLACKFIRE_SERVER_TOKEN --value <insert your Server Token>
-```
-
-### 4. Confirm it's running
-
-Login via SSH to your container and confirm that Blackfire is running as follows:
-
-```text
-php --ri blackfire | head
-
-blackfire
-
-Blackfire => enabled
-Blackfire => 1.64.0~linux-x64-zts80
-Timing measurement => cgt
-Sessions support => enabled
-Num of CPU => 8
-Profiling heap memory => 0 Kb
-Main instance trigger mode => CLI autotriggered
-
-```
-
-{{< note >}}
-Make sure you use 2 dashes in the command above. `--ri` will `Show configuration for extension <name>.`
-But `-ri` is something completely different since it will be expanded into `-r` and `-i` (run code and show full extension list).
-{{< /note >}}
+### 2. Enable Blackfire
 
+Follow [the step-by-step instructions](https://blackfire.io/docs/integrations/paas/platformsh) to enable Blackfire on your PHP or Python applications.
 
 ## On a Dedicated cluster
```

Comment:
Because one of the key features of Gen3 lets users set their env variable themselves, without having to ask for the PSH support team to do it.

Reply:
I have removed the install guide from Platform's documentation to avoid mismatched duplicates. We may have to update Blackfire's documentation in this case.

Reply:
What's key is to verify how Platform.sh deals with parallel docs depending on the "Gen" version customers use.

Reply:
I am not seeing any parallel documentation for gen3. But I am not sure to follow your thoughts and understand what to look for.